Source/WebCore/html/RangeInputType.cpp

```diff
@@ -1,6 +1,6 @@
 /*
- * Copyright (C) 2010 Google Inc. All rights reserved.
- * Copyright (C) 2011-2018 Apple Inc. All rights reserved.
+ * Copyright (C) 2010-2014 Google Inc. All rights reserved.
```

Comment:
Haven't Blink forked in 2013?

Reply:
Yes - I update Copyright year based on the year of patch, we are merging, since it was in 2014, so I update it to 2014. I checked this with @rniwa in the past. :-)